client/app/templates/account.jade

```diff
@@ -102,6 +102,13 @@
                         .account-form-error-text= t('account 2fa error')
 
             #2fa-infos.account-field
+                p#2fa-strategy
+                
+                p.bold= t('account 2fa rectokens')
```

Comment:
Yeap, I was pretty sure of it, just a little joke about those markup that we'll certainly refactor some day :smile: